crates/ra_syntax/src/grammar.ron

```diff
@@ -313,6 +313,7 @@ Grammar(
                 "DocCommentsOwner",
                 "TypeAscriptionOwner",
             ],
+            options: [ ["body","Block"]],
```

Comment:
This doesn't work because ra's syntax trees for consts / statics have a hierarchy like `ConstDef` -> `BlockExpr` -> `Block` (analogously `StaticDef`) as opposed to `FnDef` -> `Block`. Hence, the entire code fails to discover the const / static body and doesn't infer it.

Reply:
Ah no wonder nothing was showing up. I'll fix this tomorrow and squash my commits